components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +436,20 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        if (StringUtils.isBlank(reCaptchaScoreThreshold)) {
```

Comment:
what happen when someone try with v2? they don't need to configure threshold value

Reply:
so don't throw exception if it is blank or null. Can you test the recaptchav2 without threshold value